DatadogRUM/Tests/RUMMonitor/Scopes/RUMApplicationScopeTests.swift

```diff
@@ -25,33 +37,27 @@ class RUMApplicationScopeTests: XCTestCase {
         XCTAssertNil(scope.context.activeUserActionID)
     }
 
-    func testWhenFirstEventIsReceived_itStartsNewSession() throws {
+    func testWhenInitialized_itStartsNewSession() throws {
```

Comment:
/suggestion shouldn't it be 2 tests instead of a replacement? to keep testing [this statement](https://github.com/DataDog/dd-sdk-ios/pull/1542/files#diff-2a0d3031da8cdd2c56293e0221ce7a4a889ca70bf0c90f9ffad664d0e7cb1559R66).

Reply:
Rationale here is: there is no such case (in production code) where initial session is started with first interaction - after https://github.com/DataDog/dd-sdk-ios/pull/1278 we always start it with SDK init. This test was covering non-existing behaviour, hence I aligned it with reality. How does it sound?

Reply:
Sounds fair, but we are in `RUMApplicationScope`'s unit tests and not in the RUM/Monitor tests. IMHO, we could keep the test and not assume behaviours from upper layers. But it's really minor and I'm fine keeping this change 👍

Reply:
I'm gonna keep it then 👍. My additional rationale is: we want `RUMApplicationScope` tests reflect existing production logic, so when changing the implementation later (as part of Session Standarization project) tests will guard behaviours that must be supported. Keeping "first event starts session" would be a false guard in that sense.